x-pack/plugins/monitoring/server/lib/cluster/flag_supported_clusters.js

```diff
@@ -51,11 +51,12 @@ async function findSupportedBasicLicenseCluster(req, clusters, kbnIndexPattern,
  * Flag clusters as supported, which means their monitoring data can be seen in the UI.
  *
  * Flagging a Basic licensed cluster as supported when it is part of a multi-cluster environment:
- * 1. Detect if there are multiple clusters
- * 2. Detect if all of the different cluster licenses are basic
- * 3. Make a query to the monitored kibana data to find the "supported" cluster
- *    UUID, which is the cluster associated with *this* Kibana instance.
- * 4. Flag the cluster object with an `isSupported` boolean
+ * 1. Detect if there any unlinked clusters and ignore those for these calculations as they are auto supported
```

Comment:
Nit, so feel free to ignore:

From what I can tell the code in this `flagSupportedClusters` function doesn't explicitly set the unlinked cluster's `isSupported` property to `true`. I assume this works because you're handling the "supported-ness" logic for the unlinked cluster elsewhere. You might consider setting the flag for the unlinked cluster here, just so all the logic about which clusters should be supported or not is in one place.